Android/PetEver/app/src/main/java/com/example/petever/oobe/NameActivity.java

```diff
@@ -39,7 +39,8 @@ public class NameActivity extends AppCompatActivity {
     private SharedPreferences pf;
 
     private void hideKeyboard() {
-        InputManager.hideSoftInputFromWindow(getCurrentFocus().getWindowToken(), InputMethodManager.HIDE_NOT_ALWAYS);
+        InputManager.hideSoftInputFromWindow(this.getWindow().getCurrentFocus().getWindowToken(),
```

Comment:
Could you rebase based on master?
This change is conflicted with this PR(https://github.com/Pet-Ever/petever/pull/42).